File: app/src/main/java/br/com/wcisang/alojinha/ui/fragment/AboutFragment.java
package br.com.wcisang.alojinha.ui.fragment;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import br.com.wcisang.alojinha.R;

/**
 * Created by WCisang on 09/06/2018.
 */
public class AboutFragment extends Fragment {

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View v = inflater.inflate(R.layout.about_fragment, container, false);
        return v;
    }
}

File: lodjinha/app/src/main/java/com/danilodequeiroz/lodjinha/about/AboutFragment.kt
package com.danilodequeiroz.lodjinha.about

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.danilodequeiroz.lodjinha.R

val ABOUT_FRAGMENT_TAG = AboutFragment::class.java.name

class AboutFragment : Fragment() {

    override fun onCreateView(
            inflater: LayoutInflater, container: ViewGroup?,
            savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_about, container, false)
    }

}